packages/components/components/Wrapper/Wrapper.tsx

```diff
@@ -0,0 +1,17 @@
+import { styled } from "../../stitches.config";
+import React from "react";
+
+export default function Wrapper({
```

Comment:
Seems like every component that used this on the website was also wrapped in some `section` component that I could see being added here

Reply:
I lean towards just going with what you currently have considering adding `section` makes some assumptions around semantics.